src/resolvers/index.ts

```diff
@@ -180,6 +202,61 @@ const resolvers = {
         { db }
       )
     },
+    addWeatherWidget: async (
+      _: undefined,
+      { zipcode, title }: AddWeatherWidgetInput,
+      { db, user, keystoneUrl }: PortalUserContext
+    ) => {
+      if (!user) {
+        throw new AuthenticationError(
+          'You must be logged in to perform this operation'
+        )
+      }
+      let coords
+
+      // Get data from NWS to save with widget
```

Comment:
I originally had a separate resolver for `getWeatherCoords` and while it may have been more readable in this file, it really complicated things elsewhere in the app. There's potential for a future refactor if we see ourselves needing to get the lat/long separately for some reason, but I decided to stick with the simpler implementation to get this moving forward,